CONTRIBUTING.md

```diff
@@ -44,44 +74,52 @@ docker-compose -f ./docker-compose.local.yml up -d
 npm run dev
 ```
 
-## Workflow for the contributor
+== Submitting a Pull Request
 
 1.  Find the task in [JIRA](https://jira.percona.com/issues/?jql=project+%3D+PMM+AND+component+%3D+%22Grafana+Dashboards%22) or create a new task. Use a component filter Grafana Dashboards.
 
 2.  You need to make a fork of our repository in your GitHub account.
 
 3.  Make a clone of your repository on your computer.
 
-4.  Switch the repository to the correct branch. Now it's PMM-2.0.
+4.  Create a new branch for your task. Name it using the following pattern: [JIRA_ISSUE_ID]-[username]-[short_title]. For example: PMM-5053-dbazhenov-tooltip .
```

Comment:
we can relax branch naming, it is up to ppl how to call hem, we could recommend as best practice to mention issue number or just short desc